### PR TITLE
Fixes #247 : Initialise camera of the map fragment

### LINKED ISSUE #247: Initialise camera of the map fragment
**Actual Behaviour**

On opening the map fragment, the camera is initialised to the map of the world.

**Expected Behaviour**

The camera of the map should be initialised to the city location of the user which is most likely where he must have clicked the most photos.

**Screenshots of the issue**

screenshot_2017-03-28-02-44-23-424_vn mbm phimp me

**Would you like to work on the issue?**

Yes, I would like to work on it.

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #247

Screenshots for the change: 

screenshot_2017-03-28-21-17-22-619_vn mbm phimp me

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @rohanagarwal94 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/rohanagarwal94/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 1]
```diff
@@ -163,5 +206,131 @@ public boolean onMarkerClick(Marker arg0) {
...
+    }
+
+    @Override
+    public void onConnectionSuspended(int i) {
+
+    }
+
+    @Override
+    public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Can you add return statements. Blank lines are not looking good.

Comment (rohanagarwal94) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: @Abhi2424shek these methods implements Connection.callbacks methods. They don't need a return statement. For reference see open-event-android map activity. @pa1pal views?

Comment (rohanagarwal94) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: @pa1pal I think that is added when the warning is thrown at compile time for a given method. However there is no such warning. Hence I feel there is no need for that.

Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: okay

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Then remove the unused function @rohanagarwal or log it.

**Comment by rohanagarwal94** [comment]
@Abhi2424shek @pa1pal made the changes, please review.

**Comment by mariobehling** [comment]
Thank you. Let's try out a bit how it works. Cheers.

**Pull request merged**